CHANGES.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+# 0.3.2
+
+- Updated build actions to format the results dropping all ansi characters and to return error results also.
```

Comment:
You're dropping all ANSI escape sequences. If you were dropping all ANSI characters, there would be no text :)